lib/Provider/PrivacyIDEAProvider.php

```diff
@@ -294,10 +386,8 @@ public function verifyChallenge(IUser $user, string $challenge): bool
 	 */
 	private function createPrivacyIDEAInstance(): ?PrivacyIDEA
 	{
-		$this->log('info', 'Creating privacyIDEA instance...');
 		if (!empty($this->getAppValue('piURL', ''))) {
```

Comment:
`$this->getAppValue('piURL', '')` should only be called once in this function.

Reply:
[9fd6967](https://github.com/privacyidea/privacyidea-nextcloud-app/pull/26/commits/9fd696768abe1566ecc7e58e61695c8fb1bed4b3)